index.html

```diff
@@ -0,0 +1,157 @@
+      <li><a href="./about.html#event">Event</a></li>
+      <li class="accronym-link">
+        <a href=""><span> QWC</span></a>
+      </li>
+    </ul>
+  </nav>
+  <header id="headline">
+    <nav class="toolbar-container">
+      <div class="menu-btn">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+    <h2 class="slogan">"Get Ready for World cup."</h2>
+    <h1 class="title">
+      Qatar world cup <span class="sub">Opening Event</span>
+      <span class="sub">2022</span>
+    </h1>
+    <p class="intro">
+      <strong>QWC 2022 </strong> , invites you for an opening ceremony to celebrate football on and off the pitch at the
+      central fan destination in Doha-Qatar where history will be made on 16th NOvember 2022. <strong>32</strong>
+      countries will compete for the trophy.
+    </p>
+    <h3 class="event-date">2022.11.16(mon) ~ 19(wed)</h3>
+    <h4 class="event-location">@ Al Bayt Stadium | Doha |</h4>
+  </header>
+  <main>
+    <section class="program-section" id="program">
+      <h2 class="sec-title">
+        main program
+        <span class="indicator"></span>
+      </h2>
+      <div class="cards">
+        <div class="card">
+          <img src="icons/microphone.png" alt="speaker-icon" />
+          <h3 id="artiste">Artistes Performing</h3>
+          <p id="musicians">
+            Musicians from different continents will entertain 
+          </p>
+        </div>
+        <div class="card">
+          <img  src="icons/speech.png" alt="speech" /><br>
+          <h3 id="speech" >Speech</h3>
+          <p id="talk">
+            Speech from different football legends of world cup, and from the main speaker(Qatar president)
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/trophy.png" alt="trophy" />
+          <h3 id="trophy">WorldCup Trophy</h3>
+          <p>
+            The trophy that will be competed for by 32 countries will be unveiled by Qatar President.
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/sponsors.png" alt="Sponsors" />
+          <h3 id="sponsor">Sponsors</h3>
+          <p id="sponsors">
+            Sponsors for this edition of worldcup will be unveiled by the FIFA president.
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/tv.png" alt="tv" />
+          <h3 id="tvs">TV broadcast</h3>
+          <p id="tv">
+            All televisions that will broadcast all matches will be unveiled by the FIFA president.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="primary-btn">
+        Don't miss the World's <br> Entertaining Football
+      </button>
+    </section>
+    <section class="speakers-section" id="speakers">
+      <h2 class="dummy-heading">
+        This heading is used for linter pass only
+      </h2>
+      <!-- Generated dynamically -->
+      </section>
```

Comment:
Kindly consider reducing the length of words for the titles and descriptions with a possible increase in the size of the image so that the two columns they contain will remain in proportion and not differ as much. That will match the design better.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190403472-b7e08646-1828-47b1-8c0a-cd415b73862e.png)

Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/190403520-2480c23a-2ec0-438b-8091-21b65648a7e1.png)

Reply:
Reduced the length of  description  and title